cli/integration_tests/basic_monorepo/profile.t

```diff
@@ -4,6 +4,6 @@ Setup
 
 Run build and record a trace
 Ignore output since we want to focus on testing the generated profile
-  $ ${TURBO} build --profile=build.trace > turbo.log
```

Comment:
I was thinking we just rename the `SHIM` variable to `TURBO` and then remove the go binary entirely.